src/project_workspace.ts

```diff
@@ -18,24 +18,31 @@ export default class ProjectWorkspace {
   rootPath: string;
 
   /**
-   * The path to Jest, this is normally a file path like
-   * `node_modules/.bin/jest` but you should not make the assumption that
-   * it is always a direct file path, as in a create-react app it would look
-   * like `npm test --`.
+   * The command to execute Jest on the command line, this is normally a file path like
+   * `node_modules/.bin/jest` but you should not make the assumption that it is always a direct
+   * file path, as in a create-react app it would look like `npm test --`.
    *
    * This means when launching a process, you will need to split on the first
    * space, and then move any other args into the args of the process.
    *
    * @type {string}
    */
-  pathToJest: string;
+  jestCommandLine: string;
 
   /**
-   * Path to a local Jest config file.
+   * @deprecated please use `jestCommandLine` instead.  If both settings are provided, only
+   * `jestCommandLine` will be used.
+   *
+   * @type {string?}
+   */
+  pathToJest?: string;
```

Comment:
instead of maintaining both `jestCommandLine` and `pathToJest` internally and separately, what do you think if we just have shadow accessors (getter/setter) for `pathToJest` that backed by the one and only `jestCommandLine` property in the `ProjectWorkspace`. All codes/types can then just move to adopt the new name without checking both properties...

Reply:
Yeah that sounds good. Getters and setters didn't occur to me for use in JS. I think it will solve this problem, maintaining the constructor as is.  Will have a look at it.

Reply:
you can even consider adding some warning messages in the getter/setter to remind people to migrate...

Reply:
Do you mean `console.warn(...)`?

Reply:
yes